performance-test/support-files/language_code.csv

```diff
@@ -0,0 +1,4 @@
+languageCode3
+spa
+fra
+ara
```

Comment:
Is this matchaing a typical performance test env setup, cellbox supports what all languages ?

Reply:
mosip.supported-languages=eng,ara,fra
Will change accordingly.

Reply:
Reviewed testdata with updated language codes.

Reply:
I can see the eng is not in the list

Reply:
Added 'eng' to the list

Reply:
Reviewed. 'eng' lang code has been added.

Reply:
But spa is not part of supported languages right ?

Reply:
Changed the values to mosip.supported-languages=eng,ara,fra